xml/art_sle_ha_install_quick.xml

```diff
@@ -788,6 +794,139 @@ softdog                16384  1</screen>
        <keycap function="control"/><keycap>C</keycap>
       </keycombo>.
    </para>
+   </sect2>
+
+   <sect2 xml:id="sec-ha-inst-quick-test-with-cluster-script">
+    <title>Testing with the &test.script; Command</title>
+    <para>
+     The command &cmd.test.script; runs standardized tests
+     for a cluster.
+     It triggers cluster failures and verifies configuration to find
+     problems. Before you use your cluster in production, it is
+     recommended to use this command to make sure everything works as expected.
+    </para>
+    <para>
+     The command supports the following checks:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <formalpara>
+       <title>Environment Check <option>-e</option>/<option>--env-check</option></title>
+       <para>
+        This test checks:
+       </para>
+      </formalpara>
+      <itemizedlist>
+       <listitem>
+        <para> Are host names resolvable? </para>
+       </listitem>
+       <listitem>
+        <para>
+         Is the time service enabled and started?
+        </para>
+       </listitem>
+       <listitem>
+        <para>
+         Has the current node configured watchdog?
```

Comment:
I need to read the whole doc for context but I think I would suggest:
"Does the current node have a watchdog configured?"

Reply:
Yes, that sounds better. Used your suggestion.